.github/workflows/android-master.yml

```diff
@@ -0,0 +1,58 @@
+# A workflow for controlling the acions taken by CI whenever we merge code into master
+
+name: Android SDK Master CI
+
+# Triggers the workflow on when code is merged into master
+on:
+  push:
+    branches:
+      - 'master'
+
+jobs:
+  # This workflow contains a single job for running our tests
+  test:
+    name: Run Unit Tests
+
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so the job can access it
+      - uses: actions/checkout@v1
+
+      # Sets up a JDK on the job so that we can run Java code
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+
+      # Call Gradle to run the unit tests
+      - name: Unit Tests
+        run: bash ./gradlew test --stacktrace
+
+    aar:
+      name: Generate AAR
+      # The type of runner that the job will run on
+      runs-on: ubuntu-latest
+
+      steps:
+        # Checks-out your repository under $GITHUB_WORKSPACE, so the job can access it
+        - uses: actions/checkout@v1
+
+        # Sets up a JDK on the job so that we can run Java code
+        - name: Set up JDK 11
+          uses: actions/setup-java@v1
+          with:
+              java-version: 11
+
+        # Call Gradle to build our our debug AAR
+        - name: Build debug AAR
+          run: bash ./gradlew assembleDebug --stacktrace
```

Comment:
we probably need to setup android build tools first, this will probably fail due to missing license agreement

Reply:
Ya, I'm aware. Unfortunately I'll need to push out another PR first, just something that I had forgotten to ticket when storymapping.

Reply:
sounds good